Sources/TDS/Utilities/TDS+GeneralRules.swift

```diff
@@ -1,11 +1,14 @@
 /// Typealiases for making the translation of the TDS protocol more clear
 /// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/d2ed21d6-527b-46ac-8035-94f6f68eb9a8
 
+/// BIT
+public typealias Bit = UInt8
```

Comment:
We cannot get this granular, as a `BIT` truly represents only 1 bit, not 8 as is stored in a `UInt8`. This would mean something like `byteBuffer.readInteger(as: Bit.self)` would actually read an entire byte of data, which would be undesirable, so this can be removed.

Reply:
thanks for the explanation.